tests/yatai/client/test_bento_repository_api.py

```diff
@@ -227,65 +130,49 @@ def test_pull_with_labels(yatai_server_container, example_bento_service_class):
     assert labels['abc'] == '123'
 
 
-@pytest.mark.skipif(
-    sys.platform == "darwin",
-    reason="Requires docker, skipping test for Mac OS for Github Action",
-)
-@pytest.mark.skipif('not psutil.POSIX')
-def test_get(yatai_server_container, example_bento_service_class):
+def test_get(local_yatai_container_url, example_bento_service_class):
     example_bento_service_class = bentoml.ver(major=2, minor=4)(
         example_bento_service_class
     )
 
-    yc = get_yatai_client(yatai_server_container)
+    yc = get_yatai_client(local_yatai_container_url)
     test_model = TestModel()
     svc = example_bento_service_class()
     svc.pack('model', test_model)
-    svc.save(yatai_url=yatai_server_container)
+    svc.save(yatai_url=local_yatai_container_url)
     svc_pb = yc.repository.get(f'{svc.name}:{svc.version}')
     assert svc_pb.bento_service_metadata.name == svc.name
     assert svc_pb.bento_service_metadata.version == svc.version
 
 
-@pytest.mark.skipif(
-    sys.platform == "darwin",
-    reason="Requires docker, skipping test for Mac OS for Github Action",
-)
-@pytest.mark.skipif('not psutil.POSIX')
-def test_list(yatai_server_container, example_bento_service_class):
+def test_list(local_yatai_container_url, example_bento_service_class):
     example_bento_service_class = bentoml.ver(major=2, minor=5)(
         example_bento_service_class
     )
-    yc = get_yatai_client(yatai_server_container)
+    yc = get_yatai_client(local_yatai_container_url)
     test_model = TestModel()
     svc = example_bento_service_class()
     svc.pack('model', test_model)
-    svc.save(yatai_url=yatai_server_container)
+    svc.save(yatai_url=local_yatai_container_url)
 
     bentos = yc.repository.list(bento_name=svc.name)
     assert len(bentos) == 7
 
 
-@pytest.mark.skipif(
-    sys.platform == "darwin",
-    reason="Requires docker, skipping test for Mac OS for Github Action",
-)
-@pytest.mark.skipif('not psutil.POSIX')
-def test_load(yatai_server_container, example_bento_service_class):
+def test_load(local_yatai_container_url, example_bento_service_class):
```

Comment:
looks like you didn't change anything in this test, why does it work before prior to this PR?

Reply:
It was 'working' previously on accident. Previously, it will attempt to save 'remotely' and the bundle ends up in the local FS path, `/tmp`. The load method was only looks at the uri path on the bento_pb, and is able to find it in `/tmp`